### PR TITLE
PHProbe part1

### PR DESCRIPTION
**Pull request by eucalvo**
Unfinished wrapper to PHProbe class

### CODE REVIEW

File: src/TankControllerLib.cpp [hunk 1]
```diff
@@ -85,6 +86,22 @@ void TankControllerLib::loop() {
...
+void TankControllerLib::serialEvent() {
+  String string = Serial.readStringUntil(13);  // read the string until we see a <CR>
+}
+
+/**
+ * This public instance function is called when there is data on the serial port(1).
+ * This the Atlas EZO pH circuit probe.
+ */
+void TankControllerLib::serialEvent1() {
+  String string = Serial1.readStringUntil(13);  // read the string until we see a <CR>
```
Comment (jgfoster) [inline src/TankControllerLib.cpp:101]: Let's modify this to move the read into the PHProbe class.

**Review by jgfoster — CHANGES_REQUESTED** [review]
Let's talk about a minor change to the API.

**Review by prestoncarman — APPROVED** [review]
LTGM

**Review by jgfoster — APPROVED** [review]
LGTM

**Pull request merged**